guides/release/components/helper-functions.md

```diff
@@ -61,7 +61,7 @@ Since the title is just the `@username` plus some extra stuff, We can replace `@
 </section>
 ```
 
-However, to get the first initial of the string, we'll need to use JavaScript. We'll write a helper function that gives us a substring from inside a string.
```

Comment:
We say the same sort of substring stuff in a sentence right at the beginning of the next section